src/Sylius/Bundle/UiBundle/DependencyInjection/Compiler/TwigComponentTagPass.php

```diff
@@ -21,16 +21,18 @@ final class TwigComponentTagPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container): void
     {
+        $defaultTemplate = $container->getParameter('sylius_ui.twig_ux.component_default_template');
+
         foreach ($container->findTaggedServiceIds('sylius.twig_component') as $id => $tags) {
             foreach ($tags as $tag) {
-                if (!isset($tag['key'], $tag['template'])) {
-                    throw new InvalidArgumentException('The "key" and "template" attributes are required for the "sylius.twig_component" tag');
+                if (!isset($tag['key'])) {
+                    throw new InvalidArgumentException('The "key" attribute is required for the "sylius.twig_component" tag');
                 }
 
                 $twigComponentService = $container->getDefinition($id);
                 $twigComponentService->addTag('twig.component', [
                     'key' => $tag['key'],
-                    'template' => $tag['template'],
+                    'template' => $tag['template'] ?? $defaultTemplate,
                     'expose_public_props' => $tag['expose_public_props'] ?? true,
```

Comment:
Cover this updated resolving in the PassTest

Reply:
Added